charm-slurmctld/actions.yaml

```diff
@@ -1,3 +1,9 @@
+show-current-config:
+  description: >
+    Display the currently used slurm.conf.
+
+    This action is best visualized with:
+    $ juju run-action slurmctld/leader --format=json --wait | jq .[].results.slurm.conf | xargs -I % -0 python3 -c 'print(%)'
```

Comment:
What do you think about changing `python3` to just python, or using the entire path?

Reply:
Some OS still think `python` is `python2`, that's why this example uses `python3` explicitly.